ogr/services/pagure/project.py

```diff
@@ -291,6 +291,19 @@ def exists(self):
         response = self._call_project_api_raw()
         return response.ok
 
+    def is_private(self) -> bool:
+        """
+        Is this repo private? (accessible only by users with granted access)
+
+        :return: if yes, return True
+        """
+        if self.service.instance_url == "https://src.fedoraproject.org":
+            # private repositories are not allowed in fedora
+            return False
+        raise NotImplementedError(
+            f"is_private is not implemented for {self.service.instance_url}"
+        )
```

Comment:
- A little hacky..;) => Can you please create an issue for the real implementation? (So we can discuss if it is possible to implement it.)
- I am not sure about the exact match of `"https://src.fedoraproject.org"`.
  - Will the instance URL always be with `https`? If you check that from packit, I am ok with merging as is. I am just not 100% sure...

Reply:
`https://src.fedoraproject.org` is the default and also used in packit service.
I agree that it's hacky, but I was not able to find good info in pagure doc for obtaining this info. But if it's important to you, I'll contact some human.

Reply:
I tried to make it a bit nicer and talked with pingou - here is the issue: https://github.com/packit-service/ogr/issues/330

Reply:
Thanks!